lib/ex_prosemirror.ex

```diff
@@ -9,10 +9,31 @@ defmodule ExProsemirror do
   ## EEx examples
 
       <%= form_for @changeset, "#", fn f -> %>
-        <%= prosemirror_input f, :title, marks: [:em] %>
-        <%= prosemirror_input f, :body, marks: [:strong, :em], blocks: [:p, :h1] %>
+        <%= prosemirror_input f, :title, type: :title %>
+        <%= prosemirror_input f, :body, blocks: [h1: false] %>
       <% end %>
 
+  Assuming we have the following in our config:
+
+  ```elixir
+  config :ex_prosemirror,
+    default: [
+      blocks: [:p, :h1, :h2],
+      marks: [:strong]
+    ],
+    types: [
+      title: [
+        marks: [:em],
+        blocks: [:h1]
+      ]
+    ]
+  ```
+
+  We will get the following results:
+
+  * The `title` input will be instantiated with block `h1` and will have the `em` mark
+  * The `body` input will be instantiated with blocks `p` and `h2` and will have the `strong` mark
+
```

Comment:
Maybe you can write a usage sample 
E.g.

---

## Usages
```elixir
<%= prosemirror_input f, :title, type: :body ... %>
```

Reply:
hows that?